examples/twap.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const { typicalPrice, kline, TWAP } = require('..')
+
+const primitiveData = ['10002', '10001', '10000']
+const complexData = [
+  { price: '10003', ts: 1578499254526 },
+  { price: '10001', ts: 1578499255526 },
+  { price: '10000', ts: 1578499256526 }
+]
+
+console.log(kline(primitiveData)) // { close: '10002', high: '10002', low: '10000', open: '10000' }
```

Comment:
hm, very strange. why does it extrapolate values out of incomplete data? we must have all ticker values anyway, there should be hopefully no need for this?

Reply:
makes sense now. we use it on our grouped values by interval.